src/NovelRT/Timing/StepTimer.cpp

```diff
@@ -7,9 +7,7 @@
 
 namespace NovelRT::Timing {
   StepTimer::StepTimer(uint32_t targetFrameRate, double maxSecondDelta) :
-    _frequency(glfwGetTimerFrequency()),
     _maxCounterDelta((uint64_t)(_frequency * maxSecondDelta)),
-    _lastCounter(glfwGetTime()),
```

Comment:
This was removed, but needs to be corrected.

Due to `glfw` initializing _after_ the StepTimer, these were getting a value of 0, so they were moved to an initializing function. However, the StepTimer should probably be made lazily instead, so that it can be called after `glfw` is initialized in `NovelRunner` properly.

Reply:
Based on this, we _could_ also move the `glfwInit()` call and its applicable error check out of `WindowingService` and make it the earliest call in `NovelRunner` to help avoid issues of this failing.